config_samples/optimism_mainnet_config_L2.json

```diff
@@ -9,13 +9,13 @@
     "github_repo": {
         "url": "https://github.com/lidofinance/lido-l2",
         "commit": "082e7eb59de63bd376b30886568813408d04f00b",
-        "repo_prefix": ""
+        "relative_root": ""
     },
     "dependencies": {
         "@openzeppelin/contracts": {
             "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
             "commit": "d4fb3a89f9d0a39c7ee6f2601d33ffbf30085322",
-            "repo_prefix": "contracts"
+            "relative_root": "contracts"
```

Comment:
Isn't it possible to remove "/contracts" from the entry key and remove `relative_root` in the same time?

Reply:
I wouldn't do this because the package is named "@openzeppelin/contracts" and can collude with "@openzeppelin/contracts-upgradable" inside the same project e.g.

Reply:
I would leave it as is because the dependency package name and to avoid confusion on the future